oauth_dropins/reddit.py

```diff
@@ -65,14 +65,14 @@ def redirect_url(self, state=None):
     reddit = praw.Reddit(client_id=REDDIT_APP_KEY,
                          client_secret=REDDIT_APP_SECRET,
                          redirect_uri=self.request.host_url + self.to_path,
-                         user_agent='oauth-dropin reddit identity checker')
+                         user_agent='oauth-dropin reddit api')
 
     # store the state for later use in the callback handler
     models.OAuthRequestToken(id=state,
                              token_secret=state,
                              state=state).put()
     st = util.encode_oauth_state({'state':state,'to_path':self.to_path})
-    return reddit.auth.url(['identity'], st, 'permanent')
+    return reddit.auth.url(['identity','read'], st, 'permanent')
```

Comment:
these are oauth scopes? if so they should probably go into the `DEFAULT_SCOPE` class constant instead, and then use `self.scope` here. see eg `handlers.BaseHandler` and `handlers.StartHandler.post()` for details.

Reply:
it looks like the examples of DEFAULT_SCOPE require a string while this needs to be a list?  I hit an error when trying to use it as a list.

Reply:
right. a couple examples including multiple scopes, separated by `SCOPE_SEPARATOR`, eg:

https://github.com/snarfed/oauth-dropins/blob/9aa714eb3292f72ecd8376e0955c3bf635e71c49/oauth_dropins/google_signin.py#L58

this may be the first time we're using an API that expects the scopes as input as a list. i guess you'd still use `DEFAULT_SCOPES` and then split and pass in here with eg `self.scope.split(self.SCOPE_SEPARATOR)`. a bit awkward, but probably the most compatible way. i'm open to thoughts though!

Reply:
`self.scope.split(self.SCOPE_SEPARATOR)` did the trick.